sql/testdata/select_index

```diff
@@ -92,6 +101,13 @@ EXPLAIN (DEBUG) SELECT * FROM t WHERE a = 1 AND b > 'b'
 0 /t/primary/1/'c' NULL true
 
 query ITTB
+EXPLAIN (DEBUG) SELECT * FROM t WHERE a > 1 AND b > 'ba'
```

Comment:
It's an index scan in the sense that it constructed a key using the constraint `a > 1`. I can see this from debug info at `log.V(2)`. If I changed this to `a > 2` no keys would be scanned.